neil

```diff
@@ -255,8 +255,8 @@ options can be used to control the add-deps behavior:
   To support the dry run feature, side-effects should be described in the plan
   provided by deps-new-plan. This function's job is to execute side-effects
   using the plan to provide repeatability."
-  [{:keys [opts cmds]}]
-  (if (or (:help opts) (= cmds ["new"]))
+  [{:keys [opts]}]
+  (if (or (:help opts) (not (:name opts)))
```

Comment:
I think it's safer to write `(empty? opts)` here?

Reply:
If you agree, please follow up with another PR.

Reply:
@borkdude: Since we have `:exec-args` in `-main`, the `opts` will always have at least `:deps-file`. I think we'd have to move `exec-args` to a `def` and do something like `(= opts exec-args)`.

Reply:
Let me know if you still want me to make another PR to check against `exec-args` or if there's a better workaround.

Reply:
OK, well, I guess this is fine then.